### PR TITLE
Document workflow from Bedrocks to Acoustic, through Basket and CTMS

### PR DESCRIPTION
**Pull request by leplatrem**
This is an attempt to document the workflow of sign-up from bedrock to acoustic.

* [x] Base sequence
* [x] Figure out example payloads
* [x] Document VPN usecase

### CODE REVIEW

File: docs/diagrams/bedrock_basket_ctms.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+```mermaid
+sequenceDiagram
+    autonumber
+    actor Visitor
+    Visitor->>+mozilla.org: Visits landing page
+    activate mozilla.org
+    mozilla.org-->>-Visitor: Renders form [token]
+    deactivate mozilla.org
+
+    Visitor->>mozilla.org: Submit form [email, token]
```
Comment (bsieber-mozilla) [inline docs/diagrams/bedrock_basket_ctms.md:14]: I _believe_ this is a "subscribe" event; hitting this code in basket:
https://github.com/mozmeao/basket/blob/ba2abdecbddd0a186a716398b33b4dff0a775d73/basket/news/views.py#L468

**Review by bsieber-mozilla — APPROVED** [review]
Looking through the comments and seeing the github links was really helpful for following along with the different api calls--thanks for adding that detail. 

The diagram looks good to me.

(The current PR title is missing the origin of 'Bedrock' and the document name perhaps is missing `acoustic`; the naming as is wfm too)

File: docs/diagrams/bedrock_basket_ctms.md (hunk 1 above)
Comment (leplatrem) [reply docs/diagrams/bedrock_basket_ctms.md:14]: So that would be for the workflow of Relay where the form is posted on basket directly?

Comment (bsieber-mozilla) [reply docs/diagrams/bedrock_basket_ctms.md]: I believe this is the route they were planning on taking

File: docs/diagrams/bedrock_basket_ctms_acoustic.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# Basket Integration
+
+## VPN Waiting List
+
+```mermaid
+sequenceDiagram
+    autonumber
+    actor Visitor
+    Visitor->>+Bedrock: Visits mozila.org
```
Comment (bsieber-mozilla) [inline docs/diagrams/bedrock_basket_ctms_acoustic.md:9]:
```suggestion
    Visitor->>+Bedrock: Visits mozilla.org
```

**Comment by bsieber-mozilla** [comment]
LGTM!

**Pull request merged**